packages/fuel-gauge/src/options.test.ts

```diff
@@ -189,6 +204,20 @@ describe('Options Tests', () => {
   });
 
   it('prints struct option', async () => {
+    using launched = await launchTestNode({
+      contractsConfigs: [
+        {
+          deployer: OptionsAbi__factory,
+          bytecode: OptionsAbiHex,
+        },
+      ],
+    });
```

Comment:
```suggestion
    using launched = await launchOptionsContract();
```

Reply:
I need the wallet instance here on line 218

Reply:
The default is [two](https://github.com/FuelLabs/fuels-ts/blob/25587d4aeb4d33f2fc47b03719dc799d47a04a32/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L30) wallets so you always have a wallet instance available.

Reply:
Ohh now I see, sorry, you're right. :D